exla/c_src/exla/exla_client.h

```diff
@@ -22,27 +22,24 @@ class ExlaClient;
 
 class ExlaBuffer {
  public:
-  ExlaBuffer(std::unique_ptr<xla::PjRtBuffer> buffer,
-             bool erlang_managed = true);
+  ExlaBuffer(std::unique_ptr<xla::PjRtBuffer> buffer);
 
+  int device_id() { return buffer_->device()->id(); }
   xla::PjRtBuffer* buffer() { return buffer_.get(); }
   xla::StatusOr<ExlaBuffer*> CopyToDevice(xla::PjRtDevice * dst_device);
   xla::StatusOr<ERL_NIF_TERM> ToBinary(ErlNifEnv* env, exla::int64 size);
   xla::Status Deallocate();
 
   ~ExlaBuffer() {
-    // If the Erlang VM wants to GC, block it until the host uses it.
     // TODO: We likely want to keep the buffer as a shared pointer
     // between Erlang VM and XLA and use AcquireExternalReference
     // to notify that the buffer should be kept around until the
     // reference is released.
     // https://github.com/tensorflow/tensorflow/blob/b8eb820d6cb27cfa8ab65c40ce9a161de314533c/tensorflow/compiler/xla/pjrt/pjrt_client.h#L763-L780
```

Comment:
```suggestion
    // Theoretically this may block if a computation is running
    // but we always block the host until the computation is done.
```